lib/msal-node/src/config/ManagedIdentityId.ts

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import {
+    DEFAULT_MANAGED_IDENTITY_ID,
```

Comment:
Is this constant part of the public API? Why?

Reply:
I don't understand your question.

Reply:
resolved per our conversation